docs/blog/connected-registry.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Connected Registry Private Preview
+description: Private preview for ACR connected registry feature.
+ms.topic: post
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
+
+The connected registry feature of ACR allows you to deploy a registry on your premises and synchronize images between the ACR and your premises. It brings the container images and OCI artifacts closer to your on-premise container workloads and increases their performance and reliability.
+
+ACR connected registry can be used in conjunction with Azure IoT Edge, Azure Arc, Azure Stack as well as other edge container workloads.
+
+Right now, the feature is scheduled for limited private preview with selected customers starting on January 15th 2021. If you want to be considered for the private preview, please submit your contact details using this [form](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR1OsLxas9SdIhfyFenqqkolUMkFKMTdDSU45SFQzU0o0WUNROVAySkRINy4u).
```

Comment:
remove "right now". 
Connected registry is currently in limited preview. To request private preview access, submit requests via this form...

We should avoid dates and refer to customers as "selected"...